scripts/bootstrap-nixos.sh

```diff
@@ -260,8 +262,9 @@ function setup_luks_secondary_drive_decryption() {
 
 	echo "Cryptsetup luksAddKey will now be used to add /luks-secondary-unlock.key for the specified secondary drive names."
 	readarray -td, drivenames <<<"$luks_secondary_drive_labels"
-	for name in ${drivenames}[@]; do
```

Comment:
/dev/disk/by-label/ wasn't showing any of my internal disk for whatever reason, this instead uses the name of the crypt (either from /dev/mapper/ or from the disko.devices.disk.*.content.luks.name field in the disko config, hosts/common/disks/ghost.nix, for example) to get the normal /dev/{nvme*,sd*) path, and then uses that path to add the key to the device. I tested this, but only with one extra drive, so I don't know for sure that it works with multiple, but I don't see why it wouldn't.